docs/usage/dataclasses.md

```diff
@@ -260,8 +272,8 @@ except dataclasses.FrozenInstanceError as e:
 
 ### Use custom types
 
-Since stdlib dataclasses are automatically converted to add validation using
-custom types may cause some unexpected behaviour.
+Since stdlib dataclasses are automatically converted to add validation, using
+custom types may cause some unexpected behavior.
```

Comment:
Small diff between US/UK english syntax. I don't think it matters – but we should probably be consistent.